README.md

```diff
@@ -1,24 +1,151 @@
-# README
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Hi @HAMMAS-SALEEM ,

Thanks for the insightful suggestion. I will implement it.

Best Regards,
K. Inoulen Shanongba